src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -100,6 +112,16 @@ const LaunchAnalysisModal = ({
     ]),
     div({ style: { color: colors.danger(), overflowWrap: 'break-word' } }, [
       h(Fragment, wrappableOnPeriods(launchError))
+    ]),
+    location && locationType && div({}, [
+      'Output files will be saved as workspace data in:',
+      span({ style: { layout: 'flex' } }, [
+        h(RegionFlaggedContent, { location, locationType }, [
+          h(TooltipTrigger,
+            { content: div([p(['Cromwell will instruct its job runners to delocalize call outputs to your workspace bucket in this region.']), p(['Tasks within your workflow might additionally move or copy the data elsewhere.', 'You should carefully vet the workflows you run if region requirements are a concern.']), p(['Note that metadata about this run will also be stored in the US.'])]) },
+            [icon('info-circle', { style: { marginLeft: '0.5rem', marginRight: '0.2rem', cursor: 'pointer', color: colors.accent() } })])
```

Comment:
Right now this isn't accessible to keyboard or screen reader users. As a general rule, use the `Clickable` component or one of its derivatives to get the proper interactivity. In this case, there's an `InfoBox` component that might be a better fit here.